_data/committers.yml

```
@@ -176,6 +180,10 @@
   alias: yibocai
   affiliation: Arm
 
+- name: Alenka Frim
+  role: Committer
+  alias: alenka
```

```suggestion
  alias: alenkaf
```
... if this is the JIRA profile alias, it should be alenkaf 😊

---

Oh, in the phonebook there is `alenka` profile as committer? Oh, what a mess :D

---

Created an INFRA issue, hope they can help =) For now, just leave it as is Rok, thanks!